integration-tests/infinispan-common/src/test/java/org/apache/camel/quarkus/component/infinispan/common/InfinispanCommonServerTestResource.java

```diff
@@ -32,8 +32,9 @@ public class InfinispanCommonServerTestResource implements QuarkusTestResourceLi
     protected static final String USER = "camel";
     protected static final String PASS = "camel";
     private static final Logger LOGGER = LoggerFactory.getLogger(InfinispanCommonServerTestResource.class);
-    private static final String CONTAINER_IMAGE = ConfigProvider.getConfig().getValue("infinispan.container.image",
-            String.class);
+    private static final String CONTAINER_IMAGE = ConfigProvider
+          .getConfig().getOptionalValue("infinispan.container.image",
+            String.class).orElse("quay.io/infinispan-test/server:14.0.x");
```

Comment:
For Infinispan 15, main can be used instead of 14.0.x

Reply:
These images are based on every new commit on each branch

Reply:
I'd prefer our `main` branch to only run with stable image tags.

We have a dedicated branch & nightly CI build for testing the latest Quarkus SNAPSHOT. I could tweak it to override the image with the alternatives you mentioned.